src/theme.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+function Theme(chosenTheme, amount){
+  this.storyWords = [];
+  this.storyTheme = chosenTheme;
+  this.findThemeWords(chosenTheme, amount);
+};
+
+Theme.prototype.findThemeWords = function (chosenTheme, amount) {
+  var themeWordsToShuffle = themeWords[chosenTheme];
+  shuffleArray(themeWordsToShuffle);
+  this.storyWords = themeWordsToShuffle.slice(0, (amount));
+  return this.storyWords;
+};
+
+var themeWords = {
+  "fantasy" : ["these","are", "potato", "chocolate"],
```

Comment:
Let's use real fantasy words, since our tests are not dependent on these magic words (no pun intended) anymore.

`["Abnormal", "Abracadabra", "Adventure", "Alchemy", "Allegorical", "Allusion", "Amulet", "Apparition", "Apprentice", "Atmosphere", "Attraction", "Awe", "Beast", "Beauty", "Belief", "Berserk", "Bewitch", "Bizarre", "Black cat", "Blindfold", "Bogeyman", "Brew", "Brownies", "Captivate", "Cast", "Castles", "Cauldron", "Cave", "Chalice", "Changeling", "Characters", "Charisma", "Charming", "Chimerical", "Clairvoyant", "Clarity", "Classic", "Cliffs", "Clock", "Collapse", "Comic", "Compare", "Conjure", "Conspirator", "Creative", "Creature", "Crisis", "Crow", "Cruelty", "Crystal ball", "Curious", "Curse", "Dancing", "Daring", "Dazzle", "Deeds", "Deformity", "Delirious", "Demon", "Detect", "Detection", "Detective", "Disappearance", "Disaster", "Dose", "Dragon", "Dramatic", "Dread", "Dream", "Dwarf", "Eek", "Eerie", "Elf", "Empire", "Enchanting", "ESP", "Event", "Evil", "Experience", "Fable", "Fairy", "Fairy ring", "Fairy tale", "Familiar", "Fanciful", "Fantastic", "Fantasy", "Fascination", "Favors", "Fiction", "Fiery", "Figment", "Folklore", "Foolishness", "Forces", "Forgery", "Garb", "Gestures", "Ghost", "Giant", "Gifts", "Glimmer", "Gnome", "Goblin", "Godmother", "Gowns", "Grateful", "Graveyard", "Green", "Grimm", "Grotesque", "Hag", "Hallucinate", "Harbinger", "Helpful", "Herbs", "Heroic", "Hollow hills", "Horror", "Howls", "Humped back", "Idyll", "Illusions", "Image", "Imagery", "Imaginary", "Imagination", "Imp", "Impressive", "Improvise", "Impulse", "Incantation", "Incognito", "Informative", "Ingenious", "Inspiration", "Invisible", "Jargon", "Jaunt", "Jiggle", "Joking", "Keepsake", "Kettle", "Kidnap", "King", "Kingdom", "Lands", "Legend", "Legerdemain", "Leprechauns", "Lore", "Lucky", "Lunar", "Magic", "Magic carpet", "Magical", "Magician", "Majesty", "Malevolence", "Mask", "Medieval", "Medium", "Miracle", "Mischief", "Mischievous", "Misshapen", "Monster", "Moon", "Muse", "Musings", "Mysterious", "Mystery", "Mystical", "Myth", "Mythical", "Narration", "Nature", "Necromancer", "Necromancy", "Nemesis", "Newt", "Notion", "Oberon", "Odd", "Ogre", "Oracle", "Otherworldly", "Overpower", "Overwhelm", "Owl", "Pattern", "Perform", "Petrify", "Pixie", "Pixie dust", "Plot", "Poisonous", "Potent", "Potion", "Powder", "Power", "Prey", "Prince", "Prophet", "Protection", "Prowl", "Quail", "Quake", "Quash", "Quaver", "Queen", "Quest", "Question", "Quizzical", "Raconteur", "Rage", "Realm", "Reasoning", "Reference", "Reign", "Repel", "Reveal", "Robe", "Rule", "Sage", "Sandman", "Scare", "Scold", "Scroll", "Seeking", "Seer", "Setting", "Shaman", "Soothsayer", "Sorcerer", "Sorcery", "Specter", "Speculation", "Spell", "Spider", "Spirits", "Stars", "Story", "Substitution", "Supernatural", "Superstition", "Talisman", "Terror", "Theory", "Thrilling", "Torch", "Tragic", "Transform", "Tremors", "Tricks", "Troll", "Unbelievable", "Unexplained", "Unicorn", "Unique", "Unusual", "Valiant", "Valor", "Vampire", "Vanguard", "Vanish", "Vanquish", "Variety", "Venomous", "Version", "Vice", "Vicious", "Victim", "Visionary", "Vital", "Wail", "Wand", "Ward", "Watchful", "Weird", "Werewolf", "Western", "Whim", "Whimsical", "Whine", "Whisk", "Whispers", "White", "Wicked", "Willies", "Win", "Wince", "Wisdom", "Wish", "Witch", "Worry", "Worship", "Wrinkled", "Wrongdoing", "Xanadu", "Yearn", "Yesteryear", "Youth", "Yowl", "Zap!", "zealous", "Zigzag", "Zounds!"]`